### ISSUE TITLE
替换成luajit后，xcode编译报错

### ISSUE THREAD

**Issue by Cestlavieonrose** [Tencent/xLua#812]
之前的不是luajit，编译可以通过，为了使用高效的json，今天替换成了xluawithlibs仓库下的build/plugin_luajit 下的所有库 然后编译报了以下错误：
ld: warning: arm64 function not 4-byte aligned: _unwind_tester from /Users/user/dev/shizidev/autobuild/xcodesz/Libraries/libiPhone-lib.a(unwind_test_arm64.o)
Undefined symbols for architecture arm64:
  "_luaL_len", referenced from:
      _Lua_luaL_len_m590E8C0C6672C204978AFAFC3783E31E330F0AC9 in Bulk_Assembly-CSharp_10.o
     (maybe you meant: _Lua_luaL_len_m590E8C0C6672C204978AFAFC3783E31E330F0AC9)
  "_lua_geti", referenced from:
      _Lua_lua_geti_mDE702BF7E9E7B64E7457E23945EE359E096D5FA8 in Bulk_Assembly-CSharp_10.o
     (maybe you meant: _lua_getinfo, _Lua_lua_geti_mDE702BF7E9E7B64E7457E23945EE359E096D5FA8 )
  "_lua_seti", referenced from:
      _Lua_lua_seti_m69E4C34B32ABC83DFACDE1CE9796FDBD60F3895F in Bulk_Assembly-CSharp_10.o
     (maybe you meant: _Lua_lua_seti_m69E4C34B32ABC83DFACDE1CE9796FDBD60F3895F)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by Cestlavieonrose** [comment]
xcode版本是11.5

**Comment by Cestlavieonrose** [comment]
已经解决 luajit中 这几个方法没有了 我在LuaDLL.cs中将这几个方法都删除后编译通过

**Comment by chexiongsheng** [comment]
luajit版本不会用到_luaL_len
我感觉你是用了lua53编译的libxlua.a，然后配了一个libluajit.a

**Comment by Cestlavieonrose** [comment]
是的  CS工程还是原来的lua53的  然后就替换了一个libluajit.a  因为cs工程里有一些改动 需要花一点时间合并 所以先暂时这么处理了:)

**Comment by chexiongsheng** [comment]
你这样处理可能会有不可预料的问题。
你去下载个luajit版本的xlua的release，把plugins下的东西拷贝过去就可以了。

**Comment by Cestlavieonrose** [comment]
好的 我处理一下
